### DISCUSSION TITLE
Processing text/plain body results in Error: Unprocessable Entity

### DISCUSSION THREAD

**Discussion by Leem0sh** [fastapi/fastapi#8789] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [x] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [x] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [x] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [x] I commit to help with one of those options 👆

### Example Code

```python
@router.post(
    "/dimensions",
    response_model=DimensionResponseModel,
    dependencies=[maybe_security()],
    description="---",
    summary="---",
    tags=["Dimensions"],
    include_in_schema=True,
)
async def _(*, client_session: ClientSession = Depends(client_session_dep),
            description: str = Body(default="string", media_type="text/plain")) -> DimensionResponseModel:
    """
    Docs
    """
    parsed_payload = parse_obj_as(DimensionRequestBodyModel, {"description": description})
    async with client_session.post(settings.DIMENSIONS_URL, raise_for_status=True,
                                   json=parsed_payload.dict()) as the_response:
        logger.info(f"Getting dimensions from: {description}")
        return await the_response.json()
```

### Description

What am I trying to achieve? 
Process description as a plain text (without quotes) so I don't have to care about escaping etc. 

As you can see I'm using 
```
description: str = Body(default="string", media_type="text/plain")
```
which is telling Fastapi to have just a regular string in body  
![snip](https://user-images.githubusercontent.com/32645670/129634607-3ee9ec0e-10ed-480a-8edc-c81895222460.PNG)
which works great. But if I try to write something in and execute I get: 
![snip](https://user-images.githubusercontent.com/32645670/129634729-14455c58-15e7-44fd-93cb-dd535c6fd201.PNG)

I expect it to be bcs of the `-H 'accept: application/json' `? How can I change that? I tried to change the header manually with adding `headers={"accept": "text/plain"}` but without any change of the result. Am I missing something? I've been looking for a long time but without any result. Thank you.

### Operating System

Linux, Windows

### FastAPI Version

0.63.0

### Python Version

3.9.2

**Comment by SamuelHeath** [comment]
Hey @Leem0sh,

I think you just want to replace that `Body()` with a `Form()` which won't use `application/json`.

**Comment by YuriiMotov** [comment]
Can't reproduce this in current version (0.115.13).
Given initial code I simplified it to:
```python
from fastapi import Body, FastAPI

app = FastAPI(
    title="Test API",
    description="This app does nothing",
)


@app.post("/dimensions", include_in_schema=True)
async def _(*, description: str = Body(default="string", media_type="text/plain")):
    return {"description": description}
```

And it works without any problems
